api/src/routes/middleware.test.ts

```diff
@@ -1,6 +1,7 @@
 import middleware from './middleware';
 import { NextFunction, Request, Response } from 'express';
 import logger from '../util/logger';
+import * as jwtCheckModule from '../util/checkJwt';
```

Comment:
Nit: Verb and noun are still flipped throughout these tests. `jwtCheck` instead of `checkJwt.